meta-resin-common/recipes-support/resin-device-uuid/resin-device-uuid/resin-device-uuid

```diff
@@ -50,7 +50,7 @@ fi
 
 if [ -z "$uuid" ]; then
     echo "[INFO] resin-device-uuid: UUID missing from config file. Generating..."
-    uuid=$(openssl rand -hex 31)
+    uuid=$(HOME=/home/root openssl rand -hex 31)
```

Comment:
`$HOME` of root is usually `/root`. Do we have a reason to put it on `/home`?

Also, we shouldn't have to remember to set `HOME` every time we invoke openssl. We either have to set a global env var `HOME=/root` or set a global env var `RANDFILE=/path/to/.rnd` or compile openssl with `DEFAULT_HOME="/root"`. I think we should set the global HOME env var.